src/components/Message/Message.tsx

```diff
@@ -124,6 +125,45 @@ const prefetchImage = ({
     });
   }
 };
+export type GestureHandlerPayload = {
+  defaultHandler?: () => void;
+  emitter?:
+    | 'card'
+    | 'fileAttachment'
+    | 'gallery'
+    | 'giphy'
+    | 'message'
+    | 'messageContent'
+    | 'messageReplies'
+    | 'reactionList'
+    | 'textLink'
+    | 'textMention';
+  event?: GestureResponderEvent;
+};
+
+export type MessageGestureHandlerPayload<
```

Comment:
I think the naming here might be a bit confusing as it doesn't relate to the gesture handlers

Reply:
How about `MessageTouchableHandlerPayload`?

Reply:
Sounds good, gesture just seemed off.